src/main/java/kr/co/vcnc/haeinsa/HaeinsaTable.java

```diff
@@ -233,21 +234,21 @@ public HaeinsaResultScanner getScanner(@Nullable HaeinsaTransaction tx, HaeinsaS
         HaeinsaTableTransaction tableState = tx.createOrGetTableState(getTableName());
         NavigableMap<byte[], HaeinsaRowTransaction> rows;
 
-        if (Bytes.equals(scan.getStartRow(), HConstants.EMPTY_START_ROW)) {
-            if (Bytes.equals(scan.getStopRow(), HConstants.EMPTY_END_ROW)) {
+        if (Bytes.equals(!scan.isReversed() ?  scan.getStartRow() : scan.getStopRow(), HConstants.EMPTY_START_ROW)) {
+            if (Bytes.equals(!scan.isReversed() ? scan.getStopRow() : scan.getStartRow(), HConstants.EMPTY_END_ROW)) {
```

Comment:
Is the end row inclusive? So in the reversed case the start row won't be excluded right.

Reply:
I don't think it's inclusive. In reversed case, it's from [start, end).

Reply:
I think I'm confusing myself and can't think about it correctly right now, basically I'm wondering if we'll have an off-by-one error where we miss the final row in the case of reverse scanning.

Reply:
Hmm, let me see if I can add a test case here.

Reply:
Added test.